File: matiasmasso/Mat.NET/DTO/DTOStaffPos.cs
using System;

namespace DTO
{
    public class DTOStaffPos : DTOBaseGuid
    {
        public DTOLangText LangNom { get; set; }
        public int Ord { get; set; }

        public DTOStaffPos() : base()
        {
            this.LangNom = new DTOLangText(base.Guid, DTOLangText.Srcs.StaffPos);
        }

        public DTOStaffPos(Guid oGuid) : base(oGuid)
        {
            this.LangNom = new DTOLangText(base.Guid, DTOLangText.Srcs.StaffPos);
        }

        public static string Nom(DTOStaffPos oStaffPos, DTOLang oLang)
        {
            string retval = "";
            if (oStaffPos != null)
                retval = oStaffPos.LangNom.Tradueix(oLang);
            return retval;
        }
    }
}

File: matiasmasso/Mat.NET/DT2/DTOStaffPos.vb
Public Class DTOStaffPos
    Inherits DTOBaseGuid

    Property langNom As DTOLangText
    Property Ord As Integer

    Public Sub New()
        MyBase.New()
        _langNom = New DTOLangText
    End Sub

    Public Sub New(oGuid As Guid)
        MyBase.New(oGuid)
        _langNom = New DTOLangText
    End Sub

    Shared Function Nom(oStaffPos As DTOStaffPos, oLang As DTOLang) As String
        Dim retval As String = ""
        If oStaffPos IsNot Nothing Then
            retval = oStaffPos.langNom.tradueix(oLang)
        End If
        Return retval
    End Function
End Class
